Makefile

```diff
@@ -14,12 +14,20 @@
 
 SHELL = /bin/bash
 
-.PHONY: all check shellcheck
+PYTHON_FILES = $(shell find . -type f -name '*.py')
+
+.PHONY: all check shellcheck python pylint flake8
 .DEFAULT_GOAL := all
 
 all: check
-check: shellcheck
+check: shellcheck python
+python: pylint flake8
```

Comment:
feeling that this might be less confusing in the future as `check-python` or similar instead of raw `python`.